### ISSUE TITLE
"Illegal instruction" for every `kphp` call in Docker container

### ISSUE THREAD

**Issue by simonhamp** [VKCOM/kphp#884]
I'm running on an M1 MacBook Pro. Just tried setting up and using the [sample](https://vkcom.github.io/kphp/kphp-basics/compile-sample-script.html) script to compile and `kphp` just comes back with "Illegal instruction"

I haven't come across anyone else referencing this issue or found anything in the docs

Would love to give kphp a go

**Comment by simonhamp** [comment]
I've [installed the docker container](https://vkcom.github.io/kphp/kphp-basics/installation.html):

```shell
docker pull vkcom/kphp
```

Created [the sample PHP file](https://vkcom.github.io/kphp/kphp-basics/compile-sample-script.html) exactly as shown (named `index.php`)

Then [run the container](https://vkcom.github.io/kphp/kphp-basics/compile-sample-script.html#compile-and-run-with-docker):

```shell
docker run -ti -v ~/Projects/kphp/:/tmp/dev:rw -p 8080:8080 vkcom/kphp
```

Then from inside the container, I've tried to build the `index.php` file that is shared into the container:

```shell
kphp /tmp/dev/index.php
```

<img width="842" alt="Screenshot 2023-08-14 at 21 53 16" src="https://github.com/VKCOM/kphp/assets/31628/0fe4a8ad-345a-4362-9567-c954bd7486ba">

**Comment by Tsygankov-Slava** [comment]
```docker pull vkcom/kphp``` - this command download image KPHP (linux/amd64).
At the moment we haven't public image for ARM.
P.S. In the near future we'll make it publicly available.

**Comment by simonhamp** [comment]
Ah ok. Is it possible for me to build it manually on ARM?

**Comment by Tsygankov-Slava** [comment]
Yes, you can build an image manually, but some packages you should build from source, because we haven't public packages for ARM.

**Comment by ywisax** [comment]
Is there any plan to release an ARM image?
